src/main/java/generators/TransformSeriesGenerator.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2018. OpenLattice, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ *
+ */
+
+package generators;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.openlattice.client.serialization.SerializableFunction;
+import com.openlattice.shuttle.transformations.TransformValueMapper;
+import com.openlattice.shuttle.util.Constants;
+
+import java.util.List;
+import java.util.Map;
+
+public class TransformSeriesGenerator implements SerializableFunction<Map<String, Object>, String> {
+    private final List<TransformValueMapper> transformValueMapper;
+
+    /**
+     * Represents a generator to create a String from a combination of transformations
+     *
+     * @param transformValueMapper:   list of transformationValueMappers
+     */
+    @JsonCreator
+    public TransformSeriesGenerator(
+            @JsonProperty( Constants.TRANSFORM_VALUE_MAPPER) List<TransformValueMapper> transformValueMapper
+    ) {
+        this.transformValueMapper = transformValueMapper;
+    }
+
+
+    @Override
+    public String apply( Map<String, Object> row ) {
+        StringBuilder sb = new StringBuilder();
+        String sep = "";
+        for ( TransformValueMapper t : transformValueMapper ) {
+            String toAdd = t.apply(row).toString();
+            sb.append(sep).append(toAdd);
```

Comment:
Ah, I think this would be more of a `TransformConcatCombineGenerator`. I was thinking that each transform would be applied to the result of the previous transform in order, so basically no append step.
That way you could have something like this if you wanted to get a single column value and then change its output type to be something other than a String:
```
generator: !<generators.TransformSeriesGenerator>
  transforms:
    - !<transforms.GetColumnNamedTransform>
      colName: "count"
    - !<transforms.ToIntegerTransform>
```